lib/schema-evolution-manager/apply_util.rb

```diff
@@ -35,6 +36,24 @@ def apply!(dir)
       count
     end
 
+    def persist_pgpass(password)
```

Comment:
Maybe for api, we change this to "with_password_file(password)"

this will make sure we can cleanup the file in one central place